src/library/Uno.Material/Styles/Controls/v2/TextBlock.xaml

```diff
@@ -188,7 +188,21 @@
                 Value="45.454" />
     </Style>
 
-    <!--  Body Large  -->
+	<!--  Label ExtraSmall  -->
```

Comment:
@kazo0 Can you add a little comment to explain why we added this one please. 
[Like added there for now so 2 digits can enter properly in the circle of the badge count for Bottom TabBar to match Uno figma file]